specification/ml/_types/TrainedModel.ts

```diff
@@ -68,6 +68,8 @@ export class TrainedModelDeploymentStats {
   model_threads: integer
   /** The deployent stats for each node that currently has the model allocated. */
   nodes: TrainedModelDeploymentNodesStats
+  /** The sum of `peak_throughput_per_minute` for all nodes in the deployment. @since 8.2.0 */
+  peak_throughput_per_minute: integer
```

Comment:
From the PR the `peak_throughput_per_minute` and `throughput_per_minute` appear to be longs instead of integers.